pkg/volume/plugins.go

```diff
@@ -421,6 +421,9 @@ type VolumeHost interface {
 
 	// Returns an interface that should be used to execute subpath operations
 	GetSubpather() subpath.Interface
+
+	// Return a lister for CSIDriver
+	GetCSIDriverLister() storagelisters.CSIDriverLister
```

Comment:
This could be added to `AttachDetachVolumeHost` and a `KubeletVolumeHost`. Or maybe you could refactor this and and `CSINodeLister` into an interface that's just `CSIInformerVolumeHost` for use in both ADC and Kubelet.

Either way, this should not be added to the generic `VolumeHost` interface so that we don't have empty interface functions on expand, resize, etc. controllers that don't need this interface function.

Reply:
Okay.  Have CSIDriverLister in both AttachDetachVolumeHost and KubeletVolumeHost now.  It is no longer in VolumeHost.